ibis/bigquery/compiler.py

```diff
@@ -124,6 +127,76 @@ def _array_length(translator, expr):
     )
 
 
+def _arbitrary(translator, expr):
+    arg, where = expr.op().args
+    if where is not None:
+        arg_formatted = translator.translate(where.ifelse(arg, ibis.NA))
+    else:
+        arg_formatted = translator.translate(arg)
+    return 'ANY_VALUE({})'.format(arg_formatted)
+
+
+def _percentile(translator, expr):
+    args = expr.op().args
+    (arg, percentile_rank) = map(translator.translate, args[:2])
+    value_type = args[2]
+    command = dict(cont='percentile_cont', disc='percentile_disc')[value_type]
+    return '{}({}, {})'.format(command, arg, percentile_rank)
+
+
+def _approx_quantile(translator, expr):
+    args = expr.op().args
+    args = [translator.translate(arg) for arg in args[:2]]
```

Comment:
Why the `:2`? Is there a third argument you're leaving out on purpose?

Reply:
yes, `ignore_nulls` which is the next function down and raises `NotImplementedError` if it evaluates to `True`